docs/developer/index.rst

```diff
@@ -811,18 +980,25 @@ In this case the term ``32424.43`` is converted to an
 :py:class:`~sherpa.models.model.ArithmeticConstantModel` instance and then
 combined with the remaining model instance (``XSpowerlaw``).
 
-For those models that require the
-full set of elements, such as multiplication by a :term:RMF or a convolution
-kernel, requires creating a model that can "wrap" another
-model. The wrapping model will evaluate the wrapped model on
-the requested grid, and then apply any modifications.
-Examples include the
-:py:class:`sherpa.instrument.PSFModel` class,
-which creats :py:class:`sherpa.instrument.ConvolutionModel`
-instances,
-and the :py:class:`sherpa.astro.xspec.XSConvolutionKernel`
-class, which creates :py:class:`sherpa.astro.xspec.XSConvolutionModel`
-instances.
+For those models that require the full set of elements, such as
+multiplication by a :term:`RMF` or a convolution kernel, requires
```

Comment:
sentence borked. Not sure my suggestion fixes it though.
```suggestion
multiplication by a :term:`RMF` or a convolution kernel, this requires
```

I realize that this was text that's just reformatted and not written here.